src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -233,11 +255,11 @@ public String details() throws IOException {
     @Override
     public String signature() throws IOException {
         final String sign = new UncheckedText(
-            new IoCheckedScalar<>(
-                new ItemAt<>(
-                    // @checkstyle MagicNumber (1 line)
-                    6, new SplitText(this.transaction, ";")
-                )
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
```

Comment:
@paulodamaso same here, we can avoid the lambda here and re-wrapping of `String` into `IoCheckedScalar<String>` by changing `transaction` to `IoCheckedScalar`.